qa/L0_trace/opentelemetry_unittest.py

```diff
@@ -232,7 +292,7 @@ def _test_resource_attributes(self, attributes):
             ),
         )
 
-    def _verify_contents(self, spans, expected_counts):
+    def _verify_contents(self, spans, expected_counts, is_cancel):
```

Comment:
nit: `is_cancel` -> `is_cancelled`

and please update `Args` field bellow on L304 with a new field.